app/config/variables.php

```diff
@@ -764,7 +755,7 @@
             ],
             [
                 'name' => '_APP_FUNCTIONS_INACTIVE_THRESHOLD',
-                'description' => 'The minimum time a function must be inactive before it can be shut down and cleaned up. This feature is intended to clean up unused containers. Containers may remain active for longer than the interval before being shut down, as Appwrite only cleans up unused containers every hour. If no value is provided, the default is 60 seconds.',
+                'description' => 'The minimum time a function must be inactive before it can be shut down and cleaned up. This feature is intended to clean up unused containers. Containers may remain active for longer than the interval before being shut down, as Appwrite only cleans up unused containers every hour. If no value is provided, the default is 60 seconds. Note: The minimum inactive time is constrained by the MAINTENANCE_INTERVAL. If the maintenance interval is set to a value greater than the inactive threshold, functions may remain active for longer than the specified interval before being shut down.',
```

Comment:
```suggestion
                'description' => 'The minimum time a function must be inactive before it can be shut down and cleaned up. This feature is intended to clean up unused containers. Containers may remain active for longer than the interval before being shut down, as Appwrite only cleans up unused containers every hour. If no value is provided, the default is 60 seconds. The minimum inactive time is constrained by the `MAINTENANCE_INTERVAL`. If the maintenance interval is set to a value greater than the inactive threshold, functions may remain active for longer than the specified interval before being shut down.',
```

Reply:
done